pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
-        <version>1.0.9</version>
+        <version>2.0.0-SNAPSHOT</version>
```

Comment:
The next version of Jakarta Data is expected to be 1.1, not 2.0.

Reply:
Never mind. I was looking at the wrong place.  That is definitely not the version of Jakarta Data.